boards/nordic/nrf52840dk/src/main.rs

```diff
@@ -67,71 +122,121 @@ pub unsafe fn reset_handler() {
+                kernel::hil::gpio::InterruptValueWrapper::new(&nrf52840::gpio::PORT[43])
             )
             .finalize(),
             static_init!(
                 kernel::hil::gpio::InterruptValueWrapper,
-                kernel::hil::gpio::InterruptValueWrapper::new(&nrf52840::gpio::PORT[9])
+                kernel::hil::gpio::InterruptValueWrapper::new(&nrf52840::gpio::PORT[44])
             )
             .finalize(),
             static_init!(
                 kernel::hil::gpio::InterruptValueWrapper,
-                kernel::hil::gpio::InterruptValueWrapper::new(&nrf52840::gpio::PORT[8])
+                kernel::hil::gpio::InterruptValueWrapper::new(&nrf52840::gpio::PORT[45])
             )
             .finalize(),
             static_init!(
                 kernel::hil::gpio::InterruptValueWrapper,
-                kernel::hil::gpio::InterruptValueWrapper::new(&nrf52840::gpio::PORT[7])
+                kernel::hil::gpio::InterruptValueWrapper::new(&nrf52840::gpio::PORT[46])
             )
             .finalize(),
             static_init!(
                 kernel::hil::gpio::InterruptValueWrapper,
-                kernel::hil::gpio::InterruptValueWrapper::new(&nrf52840::gpio::PORT[6])
+                kernel::hil::gpio::InterruptValueWrapper::new(&nrf52840::gpio::PORT[47])
             )
             .finalize(),
             static_init!(
                 kernel::hil::gpio::InterruptValueWrapper,
-                kernel::hil::gpio::InterruptValueWrapper::new(&nrf52840::gpio::PORT[5])
+                kernel::hil::gpio::InterruptValueWrapper::new(&nrf52840::gpio::PORT[26])
+            )
+            .finalize(),
+            static_init!(
+                kernel::hil::gpio::InterruptValueWrapper,
+                kernel::hil::gpio::InterruptValueWrapper::new(&nrf52840::gpio::PORT[27])
+            )
+            .finalize(),
+            static_init!(
+                kernel::hil::gpio::InterruptValueWrapper,
+                kernel::hil::gpio::InterruptValueWrapper::new(&nrf52840::gpio::PORT[3])
+            )
+            .finalize(),
+            static_init!(
+                kernel::hil::gpio::InterruptValueWrapper,
+                kernel::hil::gpio::InterruptValueWrapper::new(&nrf52840::gpio::PORT[4])
+            )
+            .finalize(),
+            static_init!(
+                kernel::hil::gpio::InterruptValueWrapper,
+                kernel::hil::gpio::InterruptValueWrapper::new(&nrf52840::gpio::PORT[28])
+            )
+            .finalize(),
+            static_init!(
+                kernel::hil::gpio::InterruptValueWrapper,
+                kernel::hil::gpio::InterruptValueWrapper::new(&nrf52840::gpio::PORT[29])
+            )
+            .finalize(),
+            static_init!(
+                kernel::hil::gpio::InterruptValueWrapper,
+                kernel::hil::gpio::InterruptValueWrapper::new(&nrf52840::gpio::PORT[30])
             )
```

Comment:
I think the analog pins are mapped right here.

Reply:
Either you or I are looking a the wrong column. I think `PORT[3]` is pin `#3` (which is `P1_04`)  rather than `P0_03`.

Reply:
Well then in that case the LED is mapped to an analog pin. But I'm pretty sure the first column pin number is just a board-specific number and is not the pin mapping in the nRF52840 crate.

Reply:
@fhars can you help clarify?